packages/lab/stories/overlay/overlay.stories.tsx

```diff
@@ -14,53 +9,25 @@ export default {
   component: Overlay,
 } as Meta<typeof Overlay>;
 
-const OverlayTemplate: StoryFn<OverlayProps & UseOverlayProps> = (props) => {
-  const { placement, open, ...rest } = props;
-  const { getTriggerProps, getOverlayProps } = useOverlay({ placement, open });
-
-  return (
+const OverlayTemplate: StoryFn<typeof Overlay> = (props) => {
+  const OverlayContent = () => (
     <>
-      <Button {...getTriggerProps<typeof Button>()}>Toggle Overlay</Button>
-
-      <Overlay {...getOverlayProps(rest)}>
-        <div>
-          <h3 className="content-heading">Title</h3>
-          <div className="content-body">Content of Overlay</div>
-        </div>
-      </Overlay>
+      <h3 className="content-heading">Title</h3>
+      <div className="content-body">Content of Overlay</div>
+      {/* <CheckboxGroupExample /> */}
+      <br />
+      <Tooltip content={"im a tooltip"}>
+        <Button>hover me</Button>
+      </Tooltip>
     </>
   );
-};
-
-export const FeatureOverlay: StoryFn<OverlayProps & UseOverlayProps> = (
-  props
-) => {
-  const { placement, open, ...rest } = props;
-  const { getTriggerProps, getOverlayProps } = useOverlay({ placement, open });
 
   return (
-    <>
-      <div {...getTriggerProps()}>Overlay Anchor</div>
-
-      <Overlay {...getOverlayProps(rest)}>
-        <div>
-          <h3 className="content-heading">Title</h3>
-          <div className="content-body">Content of Overlay</div>
-        </div>
-      </Overlay>
-    </>
+    <Overlay {...props} content={<OverlayContent />}>
```

Comment:
This API, although similar to Tooltip. Is most nice when you only need to display simple content. Have you considered any other APIs?

Reply:
Josh and I spoke about 3 potential ways Overlay could be built. 
1. current implementation, similar to Tooltip
2. implementation similar to Dialog/Drawer
3. [new] implementation similar to latest Menu component 
#3 offers the most potential, for now keeping current implementation (#1) & reevaluate once #1 is ready

Reply:
December updates: Overlay refactored to use the 3rd implementation